rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/search.highlight/issue69009.yml

```diff
@@ -0,0 +1,68 @@
+setup:
```

Comment:
Let's add this setup section which means to skip the test for nodes with version < 8.14

setup:
  - skip:
      version: ' - 8.13.99'
      reason: 'check of preTags and postTags params for empty values was added in   8.14'